contrib/docker/README.md

```diff
@@ -32,7 +32,10 @@ $ wget https://raw.githubusercontent.com/dogecoin/dogecoin/master/contrib/docker
 $ wget https://raw.githubusercontent.com/dogecoin/dogecoin/master/contrib/docker/Dockerfile
 
 # Build the image & start a node
+# Use `--build-arg VERSION=x.y.z` to specify a version
```

Comment:
Does this default to the `VERSION` defined in `Dockerfile` if not provided? That might be worth mentioning here.

I thought I saw a release script or release commit that Ross (?) had to update the version string right before a release.

Reply:
Yes, by default the variable of Dockerfile, otherwise it overrides it. But actually, it won't work, hash is hardcoded.

I didn't know how I should check signature during the download, hashes seem ton be only available on `gitian.sigs` and not with released. I wasn't sure if I should use `sigs` folder, or if signatures have to be with tar files of the release.

In my mind, it's a possible improvement area, and it may improve security in the same time. I commented the Dockerfile about it.